CMakeLists.txt

```diff
@@ -229,7 +229,7 @@ endfunction ()
 function (ADD_CHECK_CXX_FLAG _CXXFLAGS _CACHE_VAR _FLAG)
     CHECK_CXX_COMPILER_FLAG("${_FLAG}" "${_CACHE_VAR}")
     if (${_CACHE_VAR})
-        set(${_CXXFLAGS} "${${_CXXFLAGS}} ${_FLAG}" PARENT_SCOPE)
+        set(${_CXXFLAGS} "${_FLAG} ${${_CXXFLAGS}}" PARENT_SCOPE)
```

Comment:
This changes it to prepend rather than append, instead of the other way around, right?

Reply:
Ugh, yes. I reused an older commit without looking at the commit message. I believe my reasoning for the commit message originally was about how you can append to cxxflags now when invoking cmake, instead of having it pretended.